modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -135,6 +138,26 @@ public ObjectEntry addObjectEntry(
 
 		long groupId = getGroupId(objectDefinition, scopeKey);
 
+		if (GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-153117"))) {
+			com.liferay.object.model.ObjectEntry newObjectEntry =
```

Comment:
This code block is duplicated. I think you can extract it to a local variable and reuse it. In that way you only need to wrap the `_addNestedObjectEntries` in the feature flag ìf`.